### ISSUE TITLE
Fix incompatibility with dplyr 0.8

### ISSUE THREAD

**Issue by shntnu** [shntnu/pertminr#131]
---------- Forwarded message ---------
From: <romain@rstudio.com>
Date: Fri, Feb 1, 2019 at 8:44 AM
Subject: cytominer and upcoming CRAN release of dplyr
To: <shsingh@broadinstitute.org>

Dear Shantanu Singh,

This is an automated email to let you know that:

* A new version of dplyr is ready to go to CRAN. dplyr is
  currently at version 0.7.8 and will become 0.8.0 upon release.

* cytominer uses dplyr and has problems with the new version.

* We plan to submit dplyr to CRAN on February 8.

This release represents about 9 months of development, detailed in
this blog post:
https://www.tidyverse.org/articles/2018/12/dplyr-0-8-0-release-candidate/

I need your help to keep cytominer and dplyr working together
smoothly. In the next days, can you please:

1. Read about the changes to dplyr at
   https://github.com/tidyverse/dplyr/blob/master/NEWS.md#dplyr-080.
   This page includes a list of breaking changes, the reasoning behind
   them, and to how to update your code.

2. Carefully inspect the failing checks listed at the bottom of this email.

3. For each failing check, either update your package, or tell me
   that I have a bug. If you have made changes to your package, please
   submit an update to CRAN before February 8.

If you have discovered a bug in dplyr, please file an issue (ideally
with a small reprex that illustrates the problem) at
https://github.com/tidyverse/dplyr/issues. If you're not sure whether
or not you've found a bug, please an issue and we'll help you figure
it out. Breaking changes that are not listed qualify as bugs.

Please respond to this message if you have any questions.

Thanks,

Romain Francois

== CHECK RESULTS ========================================

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      35: lang_partial_eval(call, vars, env)
      36: eval_bare(call, env)

      ══ testthat results
═══════════════════════════════════════════════════════════
      OK: 17 SKIPPED: 0 FAILED: 5
      1. Error: `aggregate` aggregates data (@test-aggregate.R#15)
      2. Error: `count_na_rows` returns the frequency of NAs per
variable (@test-count_na_rows.R#13)
      3. Error: cytominer can process dataset with a normalized schema
(@test-cytominer.R#71)
      4. Error: `drop_na_columns` removes columns have only NAs
(@test-drop_na_columns.R#11)
      5. Error: `normalize' normalizes data (@test-normalize.R#49)

      Error: testthat unit tests failed
      In addition: Warning message:
      call dbDisconnect() when finished working with a connection
      Execution halted
    ```

**Comment by gwaybio** [comment]
Tried addressing #132 but it looks like CRAN removed cytominer! (rude)

From https://cran.r-project.org/web/packages/cytominer/index.html

```
Package ‘cytominer’ was removed from the CRAN repository.

Formerly available versions can be obtained from the archive.

Archived on 2019-05-08 as check problems were not corrected in time.
```

To address #132 this must be fixed first.

**Comment by shntnu** [comment]
Fixed in #135
